recipes/area-detector-handlers/meta.yaml

```diff
@@ -25,6 +25,7 @@ requirements:
     - h5py
     - pandas
     - tifffile >=2020.8.25
+    - entrypoints
```

Comment:
@kbeeperez, we try to keep the list of dependencies in the alphabetical order (besides `python`, which goes first). Could you please move that package after `dask`?